editors/sc-ide/widgets/code_editor/completion_menu.cpp

```diff
@@ -58,10 +58,7 @@ CompletionMenu::CompletionMenu(QWidget* parent): PopUpWidget(parent), mCompletio
 
 void CompletionMenu::addItem(QStandardItem* item) { mModel->appendRow(item); }
 
-void CompletionMenu::adapt() {
-    mListView->setFixedWidth(mListView->sizeHintForColumn(0));
-    resize(0, 0);
```

Comment:
maybe substitute `resize(0,0)` with `adjustSize()`? Not fundamental though: it doesn't make a difference now because `adapt()`is currently called only before the widget is shown for the first time

Reply:
@elgiano do you think we should risk adding this for 3.14.0-rc2 ? From the discussion so far it seems that the original bug only applies to a very specific environment configuration, is hard to reproduce, and I don't have full grasp of what's actually being changed by either removing the resize or changing it to `adjustSize`.